posts/Yoann/2016-11-29-hibernate-search-5.6.0.Beta4-and-5.7.0.Beta1.adoc

```diff
@@ -0,0 +1,79 @@
+= Hibernate Search 5.6.0.Beta4 and 5.7.0.Beta1 are out!
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Today we are releasing two new versions of Hibernate Search: 5.6.0.Beta4 and 5.7.0.Beta1!
+
+Version 5.6.0.Beta4 brings the https://hibernate.atlassian.net/issues/?jql=fixVersion%20%3D%205.6.0.Beta4[lastest bugfixes and previously missing features] for our experimental Elasticsearch integration.
+
+Version 5.7.0.Beta1 brings the exact same changes as 5.6.0.Beta1, but on top of the compatibility with Hibernate ORM version `5.2.x` that was introduced with 5.7.0.Alpha1.
+
+== What's new?
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-402[HSEARCH-402]: A new `async` reader strategy has been added for the Lucene indexing service, bringing performance boosts when you are okay with your queries being run on an out-of-date index (how much out-of-date is configurable).
+ * https://hibernate.atlassian.net/browse/HSEARCH-2260[HSEARCH-2260]: A new `VALIDATE` index schema management strategy has been added for Elasticsearch, allowing you to automatically check on startup that your Hibernate Search mappings are in line with the Elasticsearch mappings.
+ * Issues with `@IndexedEmbedded` in the Elasticsearch integration have be addressed: everything now works properly, with the notable exception of https://hibernate.atlassian.net/browse/HSEARCH-2389[`@IndexedEmbedded.indexNullAs`] (not to be confused with `@Field.indexNullAs`).
+ * https://hibernate.atlassian.net/browse/HSEARCH-2235[HSEARCH-2235]: You can now configure Hibernate Search to send requests to multiple Elasticsearch servers of the same cluster, enabling load-balancing. Failover is not supported yet, but https://hibernate.atlassian.net/browse/HSEARCH-2469[we'll be working on it].
+ * https://hibernate.atlassian. net/browse/HSEARCH-2360[HSEARCH-2360]: Elasticsearch projections now use source filtering, meaning the bandwidth we use for retrieving results is as low as possible.
```

Comment:
could you tone down the "as low as possible" into a form like "we greatly reduced bandwith needs" ? I actually think we can squeeze a bit more improvements in the future ;)